nix/modules/blocksense/backends/process-compose.nix

```diff
@@ -97,6 +97,7 @@ let
           };
           log_configuration = logsConfig;
           log_location = cfg.logsDir + "/reporter-${name}.log";
+          shutdown.signal = 9;
```

Comment:
Why is this needed now?

Reply:
I am wondering if this will affect the `e2e-tests`. Right now, if you start the test setup but terminate it with `CTRL-C` process-compose will not die. I hope this will have a positive effect there too